### PR TITLE
Update SDK to version 2.0.0-dev.55.0.

### PR DESCRIPTION
**Pull request by lrhn**
Remove use of deprecated constants.

### CODE REVIEW

**Review by zanderso — APPROVED** [review]
lgtm

**Comment by lrhn** [comment]
Thanks. Please publish the package when convenient.

**Comment by nex3** [comment]
Published.

File: pubspec.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 name: http_multi_server
-version: 2.0.4
+version: 2.0.5
 author: "Dart Team <misc@dartlang.org>"
 homepage: https://github.com/dart-lang/http_multi_server
 description:
   A dart:io HttpServer wrapper that handles requests from multiple servers.
 environment:
-  sdk: ">=1.13.0 <2.0.0"
+  sdk: ">=2.0.0-dev.55.0 <3.0.0"
```
Comment (kevmoo) [inline pubspec.yaml:8]: DO NOT flip to `<3.0.0` yet – we're waiting until we know the SDK is stable

**Pull request merged**